config.h.in

```diff
@@ -1,4 +1,4 @@
 #cmakedefine CLIENT_MX "@CLIENT_MX@"
 #cmakedefine CLIENT_EMAIL "@CLIENT_EMAIL@"
 #cmakedefine CATCH_ALL_LOCAL_PART "@CATCH_ALL_LOCAL_PART@"
-#cmakedefine01 DEBUG_RESPONSE
+#cmakedefine01 DEBUG
```

Comment:
Shouldn't we declare this option only if CMAKE_BUILD_TYPE is Debug?

Reply:
How would we limit this to a Debug `CMAKE_BUILD_TYPE`?